test/unit/tracks/html-track-element-list.test.js

```diff
@@ -28,31 +28,47 @@ const genericHtmlTrackElements = [{
   track: track2
 }];
 
-q.module('HTML Track Element List');
+QUnit.module('HTML Track Element List');
 
-test('HTMLTrackElementList\'s length is set correctly', function() {
-  let htmlTrackElementList = new HTMLTrackElementList(genericHtmlTrackElements);
+QUnit.test('HTMLTrackElementList\'s length is set correctly', function() {
```

Comment:
it seems to be allowed, would you like me to change it? I am not sure if eslint can pick lines like this up so that we can automatically know about them.